src/apps/routes/apps.clj

```diff
@@ -77,6 +77,17 @@
     operation."
     (ok (apps/unshare-apps current-user (:unsharing body))))
 
+  (context "/systems" []
+
+    (GET "/ids" []
+      :query [params SecuredQueryParams]
```

Comment:
I wonder if this endpoint should be displayed under a different grouping in the Swagger docs?
The `App endpoints` group might be getting cluttered...
Maybe if we rename the `Service Status Information` group, this endpoint could go in that group?